### ISSUE TITLE
Internal Server Error

### ISSUE THREAD

**Issue by zysNLP** [zhaoyingjun/chatbot#26]
运行app.py时出现这个错误，请问怎么回事？The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.
127.0.0.1 - - [07/Dec/2018 08:51:39] "GET / HTTP/1.1" 500 -
127.0.0.1 - - [07/Dec/2018 08:51:39] "GET /favicon.ico HTTP/1.1" 404 -
2018-12-07 08:52:09 - heartbeat

**Comment by lytofd** [comment]
运行你的master分支
出现TypeError: sampled_loss() got an unexpected keyword argument 'logits'
怎么解决

**Comment by zhaoyingjun** [comment]
sampled_loss()在新版本中keyword有所变更，你按照最新的Keyword对应的修改一下就好了
